tests/xfstests/partition.pm

```diff
@@ -261,6 +261,11 @@ sub format_with_options {
         format_partition($part, 'xfs', options => '-f -m crc=1,reflink=0,rmapbt=0, -i sparse=0 -lsize=100m');
         script_run("export 'XFS_MKFS_OPTIONS=-m crc=1,reflink=0,rmapbt=0, -i sparse=0 -lsize=100m' >> $CONFIG_FILE");
     }
+    # In case to test different mkfs.btrfs options
+    elsif ($filesystem eq 'btrfs' && index(get_required_var('TEST'), '4k_pagesize') != -1) {
```

Comment:
We could have a variable which specifies the extra `mkfs.btrfs` options.

Reply:
or just the extra options passed to `format_partition` regardless of fs

Reply:
To be clear, you shouldn't change the test behavior based on the test name. People do not expect the test to behave differently if they change the name.